easydock/protonation.py

```diff
@@ -1,6 +1,11 @@
+import os.path
```

Comment:
Is it used somewhere in the code?
If so, I would prefer to use `import os` to avoid possible conflicts in namespace.
If not, then we can delete this import